SU-HSDO/suhumsci
# SSH-3160: Update carousel controls placement

# READY FOR REVIEW

## Summary
This PR fixes the carousel controls placements on the traditional and colorful theme

## Urgency
low

## Steps to Test
1. Log in to hs_traditional and hs_colorful locally.
2. Visit a carousel, such as on one of the example pages below
3. Confirm the following behavior is happening:
   - On Traditional theme, dot navigation should be horizontally aligned with arrow navigation, same as it is on Colorful theme.
   - Arrow navigation should always stay at same distance from bottom right edges of the container and not jump outside.

Traditional:
https://politicalscience.stanford.edu/
Current HP: 
![1](https://github.com/SU-HSDO/suhumsci/assets/48533432/b80c67fd-819a-43c6-abc2-895f3b8f3eb8)

Mobile:
![2](https://github.com/SU-HSDO/suhumsci/assets/48533432/1fabca29-549b-46df-b834-bf6cdf5a89c9)

Colorful: 
https://religiousstudies.stanford.edu/
Current HP:
![3](https://github.com/SU-HSDO/suhumsci/assets/48533432/44cb75d9-b5b1-4fe1-bd19-4cec2e38b76b)

Mobile:
![4](https://github.com/SU-HSDO/suhumsci/assets/48533432/27931db6-0f86-4ed6-9ead-9d3770a6a6e8)


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
